sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/OfferEvaluator.java

```diff
@@ -246,7 +247,7 @@ public OfferEvaluator(
                 .getExecutor().toBuilder()
                 .setExecutorId(Protos.ExecutorID.newBuilder().setValue(""))
                 .build();
-        logger.info("Using old executor: {}", TextFormat.shortDebugString(executorInfo));
+        logger.info("Using new executor: {}", TextFormat.shortDebugString(executorInfo));
```

Comment:
How about `Using new executor derived from task <taskInfo.getName()>: <executorDebugString>`? The retrieval at L246 could be broken into two steps: picking a TaskInfo (whose name would be logged here), and then creating the ExecutorInfo with empty id from that.